polkadot-test-service/tests/build-blocks.rs

```diff
@@ -0,0 +1,66 @@
+use async_std::task::{block_on, sleep};
+use futures::{future, pin_mut, select, FutureExt as _};
+use polkadot_test_service::*;
+use service::TaskExecutor;
+use sp_keyring::Sr25519Keyring;
+use std::time::Duration;
+
+static INTEGRATION_TEST_ALLOWED_TIME: Option<&str> = option_env!("INTEGRATION_TEST_ALLOWED_TIME");
+
+#[test]
+fn ensure_test_service_build_blocks() {
+	let task_executor: TaskExecutor = (|fut, _| {
+		async_std::task::spawn(fut);
+	})
+	.into();
+	let mut alice = run_test_node(
+		task_executor.clone(),
+		Sr25519Keyring::Alice,
+		|| {},
+		Vec::new(),
+	)
+	.unwrap();
```

Comment:
Why not panic directly in `run_test_node`? We use this stuff in tests only anyway.

Reply:
:100: you're right

It's just the library itself is not actually a test so I switch my brain to no-panic mode but you're right

Reply:
done